src/redash_stmo/extract_table_names.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
```

Comment:
Let's move this into a new `redash_stmo.utils` or `redash_stmo.handlers.query_results` package maybe?